src/main/kotlin/org/jitsi/jicofo/codec/CodecConfig.kt

```diff
@@ -0,0 +1,159 @@
+/*
+ * Jicofo, the Jitsi Conference Focus.
+ *
+ * Copyright @ 2020 - present 8x8, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.jicofo.codec
+
+import org.jitsi.config.JitsiConfig
+import org.jitsi.metaconfig.config
+
+open class CodecConfig(protected val base: String, protected val name: String) {
+    protected open val enabled: Boolean by config {
+        "$base.enabled".from(JitsiConfig.newConfig)
+    }
+
+    protected open val pt: Int by config {
+        "$base.pt".from(JitsiConfig.newConfig)
```

Comment:
Using `$base` for these properties would make it difficult to grep through the code to find the usage of a property.  I almost made this comment in another PR where you used a `base` prefix, but the key itself was unique enough I figured it would be ok.  Here, though, `pt`, `enabled`, etc. aren't very unique.